Sources/ATTNVisitorService.h

```diff
@@ -0,0 +1,39 @@
+//
+//  ATTNVisitorService.h
+//  Example
+//
+//  Created by Olivia Kim on 11/23/22.
+//
+
+#import "ATTNPersistentStorage.h"
+
+
+#ifndef ATTNVisitorService_h
+#define ATTNVisitorService_h
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+extern NSString * const VISITOR_ID_KEY;;
```

Comment:
does this need to be exposed in the header? I think the implementation file can just create a static const:
`static NSString *const VISITOR_ID_KEY = @"visitorId"`